doc/setup.md

```diff
@@ -240,6 +263,30 @@ If the command outputs information about a service, it means that port 5000 is a
 6. Change users by navigating to `http://localhost:5000/become_user/<#>` where # is a number from 1-5.
    Different users have different privileges and can see more or less of the demo data.
 
+<a id="port-configuration"></a>
+## Port Configuration
+
+Skyportal uses two distinct ports in its configuration:
+
+1. Internal Port - Where the application is actually hosted
+2. Public-Facing Port - Where external users access the application
+
+If you need to use different ports (e.g., port 5000 is already in use) add the following changes to `skyportal/config.yaml.defaults` override the defaults:
+
+```angular2html
```

Comment:
I think it's not super clear that this should be edited, rather than just added to the end of the file. Maybe just a wording thing, but adding some `...` here and there and/or comments to make it clear in that template that you are basically showing locations in the whole file and not just something to copy/paste would be super nice

Reply:
I'll add some ellipses to clarify this intent!